src/pages/Account/Tabs/ModulesTab.tsx

```diff
@@ -55,19 +80,78 @@ function ModulesContent({
   );
 }
 
-type ModulesTabProps = {
+function ModulesContent2({address}: {address: string}): JSX.Element {
```

Comment:
couple UI nits:
1. we don't need to make the UI perfect, but at least when we scroll down the source code and abi, the left panel's fixed so that we don't need to scroll all the way up to select a different module to view?
2. the left panel doesn't seem to have hover-over effect to tell you its clickable.